xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogCode.xml

```diff
@@ -835,15 +835,15 @@ $!xwiki.jsx.use($blogScriptsDocumentName)##
  * @param message A text message concerning blog article publishing
  *###
 #macro(publishMessageBox $message)
-(% class="plainmessage publish-message" %)((($message)))
+(% class="plainmessage publish-message" %)((($services.icon.render('world') $message)))
 #end
 #**
  * Displays a message box with "show/hide" icon.
  * 
  * @param message A text message concerning blog article hiding
  *###
 #macro(hideMessageBox $message)
-(% class="plainmessage hide-message" %)((($message)))
+(% class="plainmessage hide-message" %)((($services.icon.render('lock_open') $message)))
```

Comment:
"lock_open" is actually the name of a "Silk" icon. You should use "unlock" instead. Actually, the allowed values are listed in the first column there: http://design.xwiki.org/xwiki/bin/view/Proposal/XWiki+Icon+Set

Thanks!